pkg/scorecard.go

```diff
@@ -150,6 +150,26 @@ func RunScorecard(ctx context.Context,
 		"repository.defaultBranch": defaultBranch,
 	}
 
+	if len(probesToRun) > 0 {
+		probeFindings := make([]finding.Finding, 0)
+		for _, probeName := range probesToRun {
+			probeRunner, err := probes.GetProbeRunner(probeName)
+			if err != nil {
+				msg := fmt.Sprintf("could not find probe: %s", probeName)
+				return ScorecardResult{},
+					sce.WithMessage(sce.ErrScorecardInternal, msg)
+			}
+			findings, _, err := probeRunner(&ret.RawResults)
```

Comment:
RawResults is empty here. It's only populated once the checks are run. So the no actual analysis is happening.

Reply:
Should be done in https://github.com/ossf/scorecard/pull/3688/commits/e519b05f2819b7f944d38da9aae53058ad2a6064 and cleaned up in https://github.com/ossf/scorecard/pull/3688/commits/ef1ab210ebc413b8db7910653c5ea62dc162267a